assets/Product.wxs

```diff
@@ -10,14 +10,16 @@
   <?define ProductName = "$(env.ProductName)" ?>
   <?define ProductGuid = "$(env.ProductGuid)" ?>
   <?define ProductVersion = "$(env.ProductVersion)" ?>
+  <?define ProductSemanticVersion = "$(env.ProductSemanticVersion)" ?>
   <?define ProductVersionWithName = "$(var.ProductName)_$(var.ProductVersion)"?>
+  <?define ProductSemanticVersionWithName = "$(var.ProductName)_$(env.ProductSemanticVersion)"?>
```

Comment:
For an example of a platform-specific construct where we should break the dash construct, see this Debian doc (7.3) on the Debian package naming vonetino: https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics.en.html